specification/cognitiveservices/data-plane/Personalizer/preview/v1.0/Personalizer.json

```diff
@@ -990,6 +1096,16 @@
     }
   },
   "parameters": {
+    "applicationId": {
+      "name": "applicationId",
+      "in": "path",
+      "description": "Application ID of the Personalizer Services.",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "client",
```

Comment:
`x-ms-parameter-location` to be `client` is kinda weird.
I think `applicationId` parameter should be in `method`...